online-shop/src/app/components/presentational/products-details-view/products-details-view.component.html

```diff
@@ -0,0 +1,34 @@
+<html>
```

Comment:
You're not supposed to declare these tags here, that's why you have the index.html at root level. A component is essentially a piece of the app.